### PR TITLE
Tweaking User Experience guidelines

### PR DESCRIPTION
**Pull request by amyblais**

### CODE REVIEW

File: source/developer/fx-guidelines.rst [hunk 1]
```diff
@@ -29,13 +29,13 @@ Fast has two parts: being "responsive" and being "quick".
 
 **Obvious**
 
-There is no such thing as "user error"--it’s always our fault as product designers. We stop problems from incorrect usage by making users aware of correct usage. Fixes could be through user interface improvements, adjustments to help text, or helping users recover when things go wrong–adding undelete, undo and rolling back for example.
+There is no such thing as "user error"--it’s always our fault as product designers. We stop problems from incorrect usage by making users aware of correct usage. Fixes could be through user interface improvements, adjustments to help text or helping users recover when things go wrong–for example, adding undelete, undo and rolling back.
```
Comment (lindy65) [inline source/developer/fx-guidelines.rst]: For better readability should we change the second sentence to read: "We prevent problems of incorrect usage by..."?

File: source/developer/fx-guidelines.rst [hunk 2]
```diff
@@ -45,7 +45,7 @@ There will be unavoidable product failures, whether through technical issues lik
 
 Features like confirmation dialogs on critical actions or temporarily saving user input in case of disconnect are examples of forgiveness.
 
-That said, forgiving does not mean allowing the product to be used incorrectly, and breaking the obvious principle. As an analogy, if someone kicks a football and misses the goal, being forgiving means giving them a chance to kick again, it does not mean counting the miss as a goal.
+That said, forgiving does not mean allowing the product to be used incorrectly, and breaking the "Obvious" principle. As an analogy, if someone kicks a football and misses the goal, being forgiving means giving them a chance to kick again, it does not mean counting the miss as a goal.
```
Comment (lindy65) [inline source/developer/fx-guidelines.rst]: For readability, should this read: "That said, forgiving does not mean allowing the product to be used incorrectly by breaking the "Obvious" principle."?

File: source/developer/fx-guidelines.rst [hunk 3]
```diff
@@ -86,15 +85,14 @@ Desktop
...
+User experience design is a discipline drawing from ergonomics, ethnography, human factors research and cognitive psychology. It's evaluated qualitatively with tools like heuristic evaluation and cognitive walkthroughs, and quantitatively through telemetry, anonymous usage data and other automated diagnostics.
 
 As with any design process, iteration, prototyping, informed critique and evaluation is vital to doing our best work. Developing the skill of informed critique in our community increases the quality of our designs and of our product.
 
-This guide outlines the levels of UX feedback in order of positive impact in order to guide contributors towards the behaviors that best  advances a fast-moving, asychronously-developed open source product.
+This guide outlines the levels of UX feedback in the order of positive impact in order to guide contributors towards the behaviors that best  advances a fast-moving, asychronously-developed open source product.
```
Comment (lindy65) [inline source/developer/fx-guidelines.rst]: "towards the behaviors that best advances..." > "towards the behaviors that best advance..."?

File: source/developer/fx-guidelines.rst [hunk 4]
```diff
@@ -124,8 +122,7 @@ UX feedback at intermediate level or higher is highly appreciated. It should be
...
 
-   A menu with 10 items without groupings seems too long. Both Apple (link) and Windows (link) user interface guidelines suggest fewer than 7 options before grouping. I'd propose we use fewer than 5. There's research showing 5 items as an amount most people can comfortably store mentally (link), and looking through other products I don't see situations where we wouldn't be able to group by 5. Also, fewer menu items, perhaps through sub-menus, should also help mobile, where ungrouped 10 item menus just look messy. I've added a PR on UX guidelines where we can discuss this as well.
-
+   A menu with 10 items without groupings seems too long. Both Apple (link) and Windows (link) user interface guidelines suggest fewer than 7 options before grouping. I'd propose we use fewer than 5. There's research showing 5 items as an amount most people can comfortably store mentally (link), and looking through other products I don't see situations where we wouldn't be able to group by 5. Also, fewer menu items, perhaps through sub-menus, should also help mobile experience, where ungrouped 10 item menus just look messy. I've added a PR on UX guidelines where we can discuss this as well.
```
Comment (lindy65) [inline source/developer/fx-guidelines.rst]: "There's research showing 5 items as an amount..." > "There's research showing 5 items as a number..."

"...ungrouped 10 item menus..." > "ungrouped 10-item menus..."

File: source/developer/fx-guidelines.rst [hunk 5]
```diff
@@ -563,21 +559,21 @@ There are several modes for this element:
...
-Currently the `/test tool <https://github.com/mattermost/mattermost-server/tree/master/tests>`_ can be used for testing Mattermost markdown versus GitHub markdown, and we have `a ticket to update the tests for Mattermost markdown specifically <https://mattermost.atlassian.net/browse/PLT-5648>`_.
+Currently the `/test tool <https://github.com/mattermost/mattermost-server/tree/master/tests>`_ can be used for testing Mattermost markdown versus GitHub markdown.
 
 -----
 
 Organization: Reduce Obvious Steps
 ----------------------------------
 
-If the action users need to perform is obvious, we should make conscious decisions to reduce some of the steps involved in that process.
+If the action a user need to perform is obvious, we should make conscious decisions to reduce some of the steps involved in that process.
```
Comment (lindy65) [inline source/developer/fx-guidelines.rst]: "If the action a user need to perform..." > "If the action a user needs to perform..."?

File: source/developer/fx-guidelines.rst (hunk 1 above)
Comment (amyblais) [reply source/developer/fx-guidelines.rst]: Thank you Lindy! I've made all the changes (this and the ones below).

**Pull request merged**